Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Reference: https://www.lpalmieri.com/posts/fast-rust-docker-builds/
 
-FROM rust:1.63 as cargo-chef-rust
+FROM rust:1.65 as cargo-chef-rust
```

Comment:
no we aren't. this was necessary because some dependency of cargo-chef was unable to compile against 1.63.

we do define the edition of the rust syntax it uses I think but afaik the compiler version is not enforced at the crate level.